tests/e2e/pageObjects/components/myRedisDatabase/add-redis-database.ts

```diff
@@ -14,8 +14,9 @@ export class AddRedisDatabase {
     // BUTTONS
     addDatabaseButton = Selector('[data-testid^=add-redis-database]');
     addRedisDatabaseButton = Selector('[data-testid=btn-submit]');
-    addDatabaseManually = Selector('[data-testid=add-manual]');
-    addAutoDiscoverDatabase = Selector('[data-testid=add-auto]');
+    addDatabaseManually = Selector('[data-testid=add-database_tab_manual]');
+    addAutoDiscoverDatabase = Selector('[data-testid=add-database_tab_software]');
```

Comment:
don't you want to split the locators to different tabs like in Insight panel?

Reply:
not yet, we'll redo this popover, so no need to do this right now